src/modules/m_dnsbl.cpp

```diff
@@ -78,7 +78,7 @@ class DNSBLResolver : public DNS::Request
 		bool match = false;
 		in_addr resultip;
 
-		inet_aton(ans_record.rdata.c_str(), &resultip);
+		inet_pton(AF_INET, ans_record.rdata.c_str(), &resultip);
```

Comment:
This is fine here because inet_aton only worked with AF_INET sockets anyway. It is also deprecated for this reason.